src/VirtoCommerce.Platform.Web/appsettings.json

```diff
@@ -152,5 +151,10 @@
         "RedisBackplane": {
             "ChannelName": "VirtoCommerceChannel"
         }
-    }
+    },
+    "PushNotifications": {
```

Comment:
I'm not sure what we need the special `PushNotifications` section in config, the one SignalR will be enough because we do not use SignalR for other logic than PushNotification. The setting   `HubUrl` also can be part of AzureSignalR settings.

I suggest using the following config structure
```JSON
 "SignalR": {
        //Possible values: RedisBackplane (default) | AzureSignalRService
        //RedisBackplane will be activated only when RedisConnectionString is set otherwise no any SignalR scaling options will be used 
        "ScalabilityProvider": "RedisBackplane",
        "AzureSignalRService": {
            "HubUrl": "http://localhost:10645/pushNotificationHub"
            "ConnectionString": "Endpoint=https://{your service name}.service.signalr.net;AccessKey={your access key};Version=1.0;"
        },
        "RedisBackplane": {
            "ChannelName": "VirtoCommerceChannel"
        }
    },
```

Reply:
1. `HubUrl` is specific for push notifications and not related ti SignalR confguration. SignalR can be used for other purposes too.
2. `AzureSignalRService` options will be loaded only if Azure SignalR service is used